src/System.ServiceModel.Duplex/tests/System.ServiceModel.Duplex.Tests.csproj

```diff
@@ -23,9 +23,6 @@
     <None Include="project.json" />
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="$(WcfSourcePkgProj)">
-      <Name>System.Private.ServiceModel</Name>
-    </ProjectReference>
```

Comment:
Based on my understanding of what @ericstj recommended, as a workaround, we need to keep the project reference to [$(WcfSourcePkgProj)](https://github.com/dotnet/wcf/blob/master/src/test.props#L4) so the tests can run against the system.private.servicemodel.dll that was just built.